app/views/moderator.py

```diff
@@ -299,12 +299,22 @@ def render_stun_verification(self, request):
             initiator__game=game,
             receiver__game=game,
             active=False)
+        questionable_stuns = []
+        
+        for tag in unverified_stuns.filter(type=Tag.KILL):
+            similar_objects = Tag.objects.filter(initiator__game=game,
+            receiver__game=game, type=Tag.KILL, receiver=tag.receiver)
```

Comment:
Hmm why do we need this again? Is it because of the stupid `active` field?

Reply:
So I added this as a way for mods to be able to see if a tag has previously been submitted to kill a player. Because it’s really dubious to submit kills on a player twice, yet we currently allow it. Realistically, we should also check to see if a tag is reported on the same player twice in 10 minutes as well, but meh I don’t want to do that right now